packages/core-data/src/queried-data/reducer.js

```diff
@@ -102,7 +102,8 @@ function removeEntitiesById( entities, ids ) {
  */
 export function items( state = {}, action ) {
 	switch ( action.type ) {
-		case 'RECEIVE_ITEMS': {
+		case 'RECEIVE_ITEMS':
+		case 'RECEIVE_ITEM_REVISIONS': {
```

Comment:
I think ideally the "queried-data" reducer shouldn't know at all what type of "items" it's dealing with, in that sense, can we avoid introducing the "revisions" action type here and make the change in the action type in the caller?

Reply:
Oh yes! That was the last thing on the list, thank you!

Done ✅